src/Html.php

```diff
@@ -848,10 +861,12 @@ public static function textarea(string $name, ?string $value = '', array $option
     {
         $options['name'] = $name;
 
-        /** @var bool $doubleEncode */
-        $doubleEncode = ArrayHelper::remove($options, 'doubleEncode', true);
+        if (isset($options['doubleEncode']) && $options['doubleEncode'] === false) {
```

Comment:
Double encode wasn't the same as encode. There should be no way to turn off encoding for textarea since tags do not make any sense inside of it.

Reply:
Encoding is not disabled.

If double encoding is disabled, it will be encoded once:

```php
if (isset($options['doubleEncode']) && $options['doubleEncode'] === false) {
    $value = self::encode($value, false);
    $options['encode'] = false;
}
```
If double encoding is enabled, the value will be encoded encoded anyway.

```php
return self::tag('textarea', (string)$value, $options);
```

since the `encode()` method has double encoding enabled by default:

```php
encode($content, $doubleEncode = true, string $encoding = 'UTF-8'): string
```